Resources/frontend/js/jquery.last-seen-products.js

```diff
@@ -1,4 +1,4 @@
-$.subscribe('plugin/swLastSeenProducts/onCreateProductImage', function (_, _, element, data) {
+$.subscribe('plugin/swLastSeenProducts/onCreateProductImage', function (_, element, data) {
```

Comment:
It is duplicate but needed. Rename it instead

Reply:
Why is it needed? There are only 3 parameters in the original event definition. Can you point me to the needed documentation why we have to duplicate the first parameter (name)? Because this is not any valid ES / JS.

Reply:
It is valid when you execute it :wink: 
![image](https://user-images.githubusercontent.com/6224096/52343518-95a96880-2a18-11e9-93e2-eafeedd6e85e.png)

Instead, checking it with tools, you should also execute the Code once :+1:

Reply:
Sorry but this is still not valid code in strict mode.
Please try adding `"use strict";`.

See https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Strict_mode

Reply:
Did you try `$.subscribe('plugin/swLastSeenProducts/onCreateProductImage', function (_, _, element, data) {});` too instead of just printing the arguments from the inside? ;-)

Reply:
That's why I said rename the second one instead :see_no_evil:

Reply:
![image](https://user-images.githubusercontent.com/6224096/52345043-917f4a00-2a1c-11e9-918f-5df9c7f06c62.png)

Reply:
> Can you point me to the needed documentation why we have to duplicate the first parameter (name)

Still did not find any info about this in the docs or I just oversaw this.
But it seems the last one is the success callback (data)?

Reply:
> That's why I said rename the second one instead

I did so now. At first I thought it was an error that it was duplicated. Sorry for that. Gave both a name now.

Reply:
You are not in strict mode ;-)

Reply:
See the other links (Babel and Closure Compiler).